extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -807,6 +814,12 @@ export class AmpA4A extends AMP.BaseElement {
       if (this.isCollapsed_) {
         return Promise.resolve();
       }
+      // If this.iframe already exists, bail out here. This should only happen in
+      // testing context, not in production.
+      if (this.iframe) {
```

Comment:
That won't help the test breakage. Both calls to layoutCallback are executed while this.iframe == null. It's only by the time that we're in this particular promise (the second time) that this.iframe != null.